specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -307,6 +307,15 @@ The following Jakarta EE Technologies were removed from the Jakarta EE Platform.
 |Jakarta EE Technology
 |Status
 
+|Embeddable EJB Container (Jakarta Enterprise Beans, Core Features 4.0, Chapter 17)
+|Removed in Jakarta EE 10
+
+|Enterprise Web Services
+|Removed in Jakarta EE 11
+
+|Entity Beans, both Container and Bean Managed Persistence (Jakarta Enterprise Beans 4.0, Optional Features, Chapters 3 - 7)
+|Removed in Jakarta EE 10
+
```

Comment:
I envisioned that the EE 11 ones would be listed first since they are the newest and then the EE 10 ones which is why I had it in the order that I had it.  Now you have an EE 10 one and then an EE 11 one and then an EE 10 one and then 3 EE 11 ones.  To me that is confusing.

Reply:
Fixed in 406fa5a.